docs/how-to/configure-update-time-range.md

```diff
@@ -0,0 +1,45 @@
+# How to configure update-time-range
+
+### Configure update-time-range
+
+To configure when the Jenkins-k8s should check for patch and automatically update, you can apply
```

Comment:
s/should check for patch and automatically update/looks up for patches to automatically update?

Reply:
Sounds more fluent. Thank you for the suggestion, fixed!